pkgs/applications/networking/cluster/kube3d/default.nix

```diff
@@ -15,15 +15,20 @@ let
 in
 buildGoModule rec {
   pname = "kube3d";
-  version = "5.4.4";
+  version = "5.4.8";
 
   src = fetchFromGitHub {
     owner = "k3d-io";
     repo = "k3d";
     rev = "v${version}";
-    sha256 = "sha256-3J25Aj/otKDCWJ+YqAsoJogU2vckZMy7fsS8XR2EMgE=";
+    sha256 = "sha256-AweoDdw1ZktRofQZgBfzsvwuA8lVS8SDbctk5HrUAyg=";
   };
-  vendorSha256 = null;
+  # the committed vendor dir is out of sync
```

Comment:
maybe not? might just be the docker/cli go workspaces bit fighting with vendored deps

Reply:
Saw your issue upstream. Why not make it a PR to upstream?
If you propose a PR, fetchpatch is a possible. On upgrades, patches are usually removed. (Likely won't be forgotten.)
I have no preference of which path you take. I just want to resolve this issue to proceed. :-)

Reply:
There are a couple solutions and I'd rather get some feedback on what fix to implement first. Rather than wasting time on an approach and having to change it.

TBH upstream golang team should have thought about this more when implementing workspaces